home.html

```diff
@@ -0,0 +1,128 @@
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>Web Conference 2021</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
+      type="text/css"
+    />
+    <link href="styles.css" rel="stylesheet" />
+  </head>
+  <body>
+    <header class="header">
+        <nav class="first-nav">
+          <i class="fa fa-times icon cursor xl-none none" id="close"></i>
+          <i class="fa fa-bars icon cursor xl-none" id="open"></i>
+            <ul class="first-nav-ul">
+                <li class="first-nav-list"><a href="#" class="first-nav-anchor"><i class="fa fa-facebook-f"></i></a></li>
+                <li class="first-nav-list"><a href="#" class="first-nav-anchor"><i class="fa fa-twitter"></i></a></li>
+                <li class="first-nav-list"><a href="#" class="first-nav-anchor">English</a></li>
+                <li class="first-nav-list"><a href="#" class="first-nav-anchor">My page</a></li>
+                <li class="first-nav-list"><a href="#" class="first-nav-anchor">Logout</a></li>
+            </ul>
+            </nav>
+        <nav class="second-nav none" id="nav">
+            <ul class="second-nav-logo">
+              <li><a href="./home.html"><i class="fa fa-first-order footer-icon"></i></a></li>
+                <li class="second-nav-logo-list">Global Tech 2021</li>
+            </ul>
+            <ul class="second-nav-ul">
+               <li><a href="./about.html" class="second-nav-anchor">About</a></li> 
+               <li><a href="#" class="second-nav-anchor">Program</a></li> 
+               <li><a href="#" class="second-nav-anchor">Join</a></li> 
+               <li><a href="#" class="second-nav-anchor">Sponsor</a></li> 
+               <li><a href="#" class="second-nav-anchor">News</a></li> 
+               <li><a href="#" class="second-nav-anchor last">Academics</a></li> 
+            </ul>
+        </nav>
+    </header>
+      <main>
+       
+          <section class="main-section">
+             <h1 class="main-header m-top">"Hello! Technology World"</h1>
+             <h2 class="sub-header"> Web development & design courses 2021</h2>
+             <p class="paragraph">
+                 A free web development and web design training with technical support, free handbook,web conference and career support from over 30 experts is taking place in october, in Nigeria
+             </p>
+             <p class="time">2021.10.15(Thu) ~ 16(fri)</p>
+             <p class="venue">@ National Stadium of Abuja, Art Center Lagos and more</p>
+          </section>
```

Comment:
- [ ] This section should have a 
![Screenshot from 2021-07-01 20-43-02](https://user-images.githubusercontent.com/1642014/124167745-f4218680-daac-11eb-8e3b-34d298b7fc46.png)
more left margin.

Reply:
:+1: Addressed successfully!